README.md

```diff
@@ -198,6 +198,9 @@ You can also load your JSONified config in from an ENV variable (e.g. `KUBE_CONF
 ```
 Kubeclient::Config.new(JSON.parse(ENV['KUBE_CONFIG']), nil)
 ```
+###Supported kubernetes versions
+
+For 1.1 only the core api is supported, apis/extensions are supported in later versions.
```

Comment:
We could in principle extend the mapping, right?  It's just we don't yet need them?

Saying 1.1 is not precise, when I tried Origin 1.1 I did get `kind` fields.  See my comment on naming.